Bloom.md

```diff
@@ -0,0 +1,248 @@
+---
+RFC: 4
+Status: Proposed
+---
+
+# ValKeyBloom Module RFC
+
+## Abstract
+
+The proposed feature is ValKeyBloom which is a Rust based Module that brings a native bloom filter data type into Valkey.
+The module should be API-compatible with Redis Ltd.’s ReBloom Module.
+
+## Motivation
+
+Bloom filters are a space efficient probabilistic data structure that can be used to “check” whether an element exists in a set (with a defined false positive), and to “add” elements to a set. While checking whether an item exists, false positives are possible, but false negatives are not possible. https://en.wikipedia.org/wiki/Bloom_filter
```

Comment:
```suggestion
Bloom filters are a space efficient probabilistic data structure that can be used to “check” whether an element exists in a set (with a defined false positive), and to “add” elements to a set.
While checking whether an item exists, false positives are possible, but false negatives are not possible. https://en.wikipedia.org/wiki/Bloom_filter
```
It makes it easier to read if paragraph lines are broken up. Either reflow at some line length (like 120 characters) or break on each period.

Reply:
That makes sense. I added line length based new line breaks at 120 characters